integrations/fathom/gitbook-manifest.yaml

```diff
@@ -36,11 +36,27 @@ summary: |
 categories:
     - analytics
 configurations:
+    space:
```

Comment:
We don't use this `space` configuration key anymore - so you can omit this one from the configuration (See `main` right now, as it's been removed

Reply:
Done